database/articles.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const expressApp = express();
+const bodyParser = require('body-parser');
+const jsonParser = bodyParser.json();
+
+let articles = [];
+
+const getAllArticles = () => {
+  return articles;
+};
+
+const postArticle = (req, res) => {
+  if (req.body) {
+    articles.push(req.body);
+    res.send({ success: true });
+    console.log(articles);
+  } else {
+    res.send({ success: false });
+  }
+};
+
+const putArticle = (req, res) => {
+  if (req.body) {
+    for (let i = 0; i < articles.length; i++) {
+      if (articles[i].title === req.body.title) {
+        articles[i] = req.body;
+      }
+    }
+    res.send({ success: true });
+    console.log(articles);
+  } else {
+    res.send({ success: false });
+  }
+};
+
+module.exports = {
+  getAllArticles: getAllArticles,
```

Comment:
Nothing wrong with this code but just FYI if your object keys and variable names are the same with ES6 you can do:
```
module.exports = {
  getAllArtices,
  postArticle,
  putArticle
}
```